lib/push.js

```diff
@@ -46,6 +54,21 @@ module.exports = function (log, db) {
     })
   }
 
+  /**
+   * Reports push increment actions to logs
+   *
+   * @param name
+   * Name of the push action
+   */
+  function incrementPushAction(name) {
+    if (name) {
+      log.info({
+        op: LOG_OP_PUSH_TO_DEVICES,
+        name: name
+      })
+    }
+  }
```

Comment:
Worth still doing the `log.increment` here as well as `log.info`, for when datadog gets more reliable?